lib/src/client.dart

```diff
@@ -121,6 +121,24 @@ class Client extends MatrixApi {
   /// The timeout until a typing indicator gets removed automatically.
   final Duration typingIndicatorTimeout;
 
+  DiscoveryInformation? _wellKnown;
```

Comment:
Do we need to cache in the database AND in memory? I would leave the memory caching up to the client developer tbh. We already have waaay to many states to handle

Reply:
I see but caching it at runtime it way more convenient since it can be accessed without async API. But however you like, I can remove it.

Reply:
yes IMO this can better be done by the app. We already have so many in-memory caches which leads to huge performance problems, that in the long term I would prefer to get rid of those and don't add more of them

@td-famedly @nico-famedly feel free to add your opinion here as well